add_tag.sh

```diff
@@ -15,14 +15,21 @@ if [ ! -f ~/.ssh/github_deploy.pem ]; then
     chmod 400 $TRAVIS_BUILD_DIR/github_deploy.pem && mv $TRAVIS_BUILD_DIR/github_deploy.pem ~/.ssh/ && ssh-add ~/.ssh/github_deploy.pem
 fi
 
-# Figure out what's the new version
-python $SCRIPT_DIR/update_version.py "$TRAVIS_BUILD_DIR/version.json" "`git describe --abbrev=0 --tags 2>&1`"
-export SA_VERSION=`cat $TRAVIS_BUILD_DIR/version.json | jq -r .application_version`
-# Set Travis fake user & email
-git config --global user.email "${GH_USER_EMAIL}"
-git config --global user.name "${GH_USER_NAME}"
-# Switch to SSH url
-git remote set-url origin git@github.com:${TRAVIS_REPO_SLUG}.git
-# Add and push tag
-git tag -a $SA_VERSION -m "Release $SA_VERSION"
-git push origin $SA_VERSION
+if [ -z "${SHOULD_UPDATE_VERSION}" ]; then
+    # Use git tag for versioning, updating package.xml in the process
+    # Figure out what's the new version
+    python $SCRIPT_DIR/update_version.py "$TRAVIS_BUILD_DIR/version.json" "`git describe --abbrev=0 --tags 2>&1`"
+    export SA_VERSION=`cat $TRAVIS_BUILD_DIR/version.json | jq -r .application_version`
+    # Set Travis fake user & email
+    git config --global user.email "${GH_USER_EMAIL}"
+    git config --global user.name "${GH_USER_NAME}"
+    # Switch to SSH url
+    git remote set-url origin git@github.com:${TRAVIS_REPO_SLUG}.git
+    # Add and push tag
+    git tag -a $SA_VERSION -m "Release $SA_VERSION"
+    git push origin $SA_VERSION
+else 
+    # Use Travis build number for versioning; Git tags & package manifest will not be updated.
+    export SA_VERSION=$TRAVIS_BUILD_NUMBER
```

Comment:
Lets pass in the Major & Minor version as parameters to this function and change: 
SA_VERSION=$MAJOR_VERSION.$MINOR_VERSION.$TRAVIS_BUILD_NUMBER